cmake/vendors/espressif/esp32.cmake

```diff
@@ -116,7 +139,13 @@ target_sources(
 afr_mcu_port(ota)
 target_sources(
     AFR::ota::mcu_port
-    INTERFACE "${AFR_MODULES_DIR}/ota/portable/espressif/esp32_devkitc_esp_wrover_kit/aws_ota_pal.c"
+    INTERFACE
+        "${AFR_MODULES_DIR}/ota/portable/espressif/esp32_devkitc_esp_wrover_kit/aws_ota_pal.c"
+        "${AFR_MODULES_DIR}/ota/portable/espressif/esp32_devkitc_esp_wrover_kit/aws_esp_ota_ops.c"
+)
+target_link_libraries(
```

Comment:
Do we need this dependency information here? OTA already depends on MQTT, which should chain to pkcs11